runner/runners/facade.go

```diff
@@ -0,0 +1,21 @@
+package runners
+
+import (
+	"github.com/scootdev/scoot/runner"
+)
+
+// NewQueueRunner creates a new Service that uses a Queue
+// (This is what the Daemon and Worker will call for now, and it will take an Execer, Filer, &c.)
+func NewQueueRunner() runner.Service {
+	controller := NewQueueController()
+	statuses := NewStatuses()
+	return &ServiceFacade{controller, statuses, statuses}
+}
+
+// ServiceFacade makes a Service from component parts.
+type ServiceFacade struct {
```

Comment:
It's the only thing that satisfies Service. So everywhere we pass around Runner today (that really needs the whole Service, not just the Controller or the StatusReader), we'd pass around a Facade.